### PR TITLE
Bumped up to API v2.6 and fixed default value count issue #11

### PR DESCRIPTION
**Pull request by katua**
- Updated the code to use API v2.6 and latest utils;
- fixed issue #11, increased the count for data reduction to 30K.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,23 @@
 {
-  "name": "text-filter",
+  "name": "visual",
```
Comment (stopyoukid) [inline package.json]: I would copy over the name

File: pbiviz.json [hunk 2]
```diff
@@ -1,33 +1,23 @@
 {
   "visual": {
     "name": "textFilter",
-    "displayName": "Text Filter",
-    "guid": "textFilter25A4896A83E0487089E2B90C9AE57C8A",
+    "displayName": "TextFilter",
```
Comment (stopyoukid) [inline pbiviz.json]: You'll need to copy over `displayName`, `guid`, `version`, `description` from the old one.

File: pbiviz.json [hunk 3]
```diff
@@ -1,33 +1,23 @@
...
     "supportUrl": "",
     "gitHubUrl": ""
   },
-  "output": "dist/textFilter.pbiviz",
-  "apiVersion": "1.11.0",
+  "apiVersion": "2.6.0",
   "author": {
-    "name": "Microsoft (experimental)",
-    "email": "msrvizsupport@microsoft.com"
+    "name": "",
```
Comment (stopyoukid) [inline pbiviz.json]: I would change "name" to "Microsoft" and copy over the old email address

File: src/settings.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (stopyoukid) [inline src/settings.ts]: We can remove this file, I don't think we are even using it

**Review by stopyoukid — CHANGES_REQUESTED** [review]
Have you run lint?

File: package.json (hunk 1 above)
Comment (katua) [reply package.json]: Updated

File: pbiviz.json (hunk 2 above)
Comment (katua) [reply pbiviz.json]: Fixed

File: pbiviz.json (hunk 3 above)
Comment (katua) [reply pbiviz.json]: Fixed

**Pull request closed without merging**